Add realm icon to AccountItems in organisation switcher, Fixes: #392

vaibhavs2 opened the pull request:
Fixes: #392
|1|2|3|
|---|---|---|
|<img src="https://user-images.githubusercontent.com/31806059/114435265-9b46fe00-9be1-11eb-91e8-d132f3b0c6b1.jpg" height="360" />|<img src="https://user-images.githubusercontent.com/31806059/114435277-9e41ee80-9be1-11eb-9ad9-a577149dd522.jpg" height="360" />|<img src="https://user-images.githubusercontent.com/31806059/114435298-a306a280-9be1-11eb-9b8f-bd225f88c5e1.jpg" height="360" />|

The third image above, I created that mannualy to show some realm icons together :)

If some wonder why I made icon container round look at the image below(not looking good), sometime we can get round icon sometime square shaped, so its better to have round container, we can adjust square shaped Icons in round container.  

<img src="https://user-images.githubusercontent.com/31806059/114435308-a4d06600-9be1-11eb-8d6d-c0eef779ea93.png" height="360" />

chrisbobbe commented:
> If some wonder why I made icon container round look at the image below(not looking good), sometime we can get round icon sometime square shaped, so its better to have round container, we can adjust square shaped Icons in round container.

I disagree; I think we'd be better off using the image with no shape change, or at most make it a square with (slightly) rounded corners. I think the basic reason is that some realm logos weren't built to look best as a circle, and we shouldn't assume they were; see #205 and #3398 where we settled on rounded-corner squares for avatar images for that reason. 🙂

vaibhavs2 committed:
realmIcon: Add realmIcon in Account-types, function parameters and tests

Passing realIcon as email and realm were used to pass so we can have them always and will store in the same way.

vaibhavs2 committed:
realmIcon: Add icon to AccountItems in AccountPickerScreen

vaibhavs2 commented:
Made corner slightly rounded :)

<img src="https://user-images.githubusercontent.com/31806059/114466488-2cc86700-9c06-11eb-9fdf-bb27613cccb1.png" height="360"/>

_again above list mannualy created_

vaibhavs2 commented:
@chrisbobbe thanks for review, now its ready for another review :)
